core/client/app/components/gh-upload-modal.js

```diff
@@ -22,6 +24,19 @@ export default ModalDialog.extend({
             this.$('.js-upload-url').removeClass('error');
         }
     },
+    addKeyboardShortcutHandler: function () {
+        var self = this;
+
+        this.$(document).on('keydown', function (event) {
+            event.preventDefault();
+            if (event.keyCode === keyboardCodes.enter) {
+                self.get('controller').send('confirm', 'accept');
+            }
+        });
+    },
+    willDestroyElement: function () {
+        this.$(document).off('keydown');
```

Comment:
We definitely don't want to remove all `keydown` handlers from the `document` element. Can you use jQuery's namespacing so that we can remove only this modal's enter-key handler?